framework/packages/abstract-interface/src/deployment.rs

```diff
@@ -306,6 +306,10 @@ impl<Chain: CwEnv> Abstract<Chain> {
             blob: self.blob.clone(),
         }
     }
+
+    pub fn account_code_id(&self) -> Result<u64, AbstractInterfaceError> {
```

Comment:
Do we want code_id method on "account" object instead? Where we load code_id for "abstract::account" from cw orch env state

Reply:
Right now it will fail because contract id is different depending on account id

Reply:
Yeah that's why I put that on Abstract. I wanted a way to get the account id that should be used for creating a new account. This function might want to load the latest code id from chain, that could be better I think